ui/hooks/ramps/useRamps/useRamps.ts

```diff
@@ -32,7 +32,7 @@ const useRamps = (
   const isMarketingEnabled = useSelector(getDataCollectionForMarketing);
 
   const getBuyURI = useCallback(
-    (_chainId: ChainId | CaipChainId) => {
+    (_chainId: Hex | CaipChainId) => {
```

Comment:
The caller of this function uses the result of calling `getCurrentChainId`, which now has a type, and  `ChainId | CaipChainId` is not compatible.